load

```diff
@@ -1,11 +1,13 @@
+#!/bin/sh
+
 #Credit: Aixxe @ aixxe.net
-if grep -q $(realpath ./libAimTux.so) /proc/$(pidof csgo_linux64)/maps; then
+if grep -q libAimTux.so /proc/$(pidof csgo_linux64)/maps; then
```

Comment:
It is not mandatory, since we don't need to search for the absolute path in the maps list, only the .so filename is sufficient